.github/workflows/merge-commits-to-next-branch.yml

```diff
@@ -0,0 +1,107 @@
+name: Merge commits to next branch
+
+on:
+  schedule:
+    - cron: "0 */2 * * *"
+
+env:
+  COMMIT_INFO: commit-info
+
+jobs:
+  dequeue:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Install Azure Storage
+        run: npm i @azure/storage-queue
+      - name: Check queue first item
+        uses: actions/github-script@v6
+        with:
+          script: |
+              const { QueueClient, QueueServiceClient } = require("@azure/storage-queue");
+              const connectionString = `${{ secrets.AZURE_STORAGE_CONNECTION_STRING }}`;
+              const queueName = `${{ secrets.AZURE_QUEUE_NAME }}`;
+              const queueServiceClient = QueueServiceClient.fromConnectionString(connectionString);
+              const queueClient = queueServiceClient.getQueueClient(queueName);
+              var receivedMessages = await queueClient.receiveMessages();
+              const message = receivedMessages.receivedMessageItems[0];
+              $COMMIT_INFO = JSON.parse(message.messageText);
+  create-branch:
+    runs-on: ubuntu-latest
+    if: ( $COMMIT_INFO.label == 'queued' )
+    steps:
+      - name: Create branch
+        uses: actions/github-script@v6
+        with:
+          script: |
+            git fetch origin
+            git checkout next
+            git branch $COMMIT_INFO.sha
+            git checkout $COMMIT_INFO.sha
+            git cherry-pick $COMMIT_INFO.sha
+  createPullRequest:
+    runs-on: ubuntu-latest
+    if: ( $COMMIT_INFO.label == 'queued' )
+    steps:
+      - name: Create Pull Request
+        uses: actions/github-script@v6
```

Comment:
Before you merge, use <https://github.com/mheap/pin-github-action> to pin the versions of the actions to a hash. It's easy to use.

Reply:
pinned GitHub action before merging this PR